atlite/wind.py

```diff
@@ -90,3 +101,105 @@ def extrapolate_wind_speed(ds, to_height, from_height=None):
+        or a manufacturer and turbine name, e.g. the following are identical:
+        {'id':10}, {'id':10, 'manufacturer':'Unknown', name:'E-53/800'},
+        {'manufacturer':'Enercon', name:'E-53/800'}
+    store_locally : bool
+        (Default: True) Whether the downloaded config should be stored locally
+        in config.windturbine_dir.
+        
+    Returns
+    -------
+    turbineconf : dict
+        The turbine configuration downloaded and stored is also returned as a dict.
+        Has the same format as returned by 'atlite.ressource.get_turbineconf(name)'.
+    """
+    
+    try: 
+        # Get the turbine list
+        OEDB_URL = 'https://openenergy-platform.org/api/v0/schema/supply/tables/turbine_library/rows'
+        result = requests.get(OEDB_URL)
+    except requests.exceptions.RequestException as e:
+        logger.info(f"Connection to OEDB failed with:\n\n{str(e)}")
+        return None
+
+    # Convert JSON to dataframe for easier filtering
+    # Only consider turbines with power curves available
+    df = pd.DataFrame.from_dict(result.json())
+    df = df[df.has_power_curve]
+
+    # Check which information is provided for lookup and proceed accordingly
+    if turbine.get('id'):
+        logger.info(f"Searching for turbine in OEDB with id '{turbine['id']}'.")
+        ds = df[df.id == int(turbine['id'])]
+        
+    else:
+        if not all({isinstance(v, str)
+                    for k,v in turbine.items() if k in ['name', 'manufacturer']}
+                  ):
+            
+            logger.error("'name' and 'manufacturer' must be provided. "
+                         "Alternatively provide a turbine id from OEDB.")
+            return None
+        logger.info(f"Searching turbine power curve in OEDB database using "
+                    f"manufacturer '{turbine['manufacturer']}'"
+                    f"and name '{turbine['name']}'.")
+            
+        ds = df[df.name == turbine['name']]
+        ds = ds[ds.manufacturer == turbine['manufacturer']]
+
+    if len(ds) < 1 :
+        logger.info(f"No turbine found for {turbine}.")
+        return None
+    elif len(ds) > 2 :
+        logger.info(f"Provided information corresponds to more than one turbine. "
+                    f"Provide id to unambiguous lookup. {turbine}.")
+        return None
+    elif len(ds) == 1:
+        # Convert to series for simpliticty
+        ds = ds.iloc[0]
+
+    # convert power from kW to MW
+    power = np.array(json.loads(ds.power_curve_values)) / 1e3
+
+    turbineconf = {
+        "name": ds['name'].strip(),
+        "manufacturer": ds.manufacturer.strip(),
+        "source": f"Original: {ds.source}. Via OEDB {OEDB_URL}",
+        "HUB_HEIGHT": ds.hub_height.strip(),
+        "V": json.loads(ds.power_curve_wind_speeds),
+        "POW": power.tolist(),
+    }
+
+    if store_locally is True:
+        filename = "{m}_{n}.yaml".format(m=turbineconf['manufacturer'],
```

Comment:
```suggestion
        filename = ("{manufacturer}_{name}.yaml".format(**turbineconf)
                    .replace("/", "_").replace(" ", "_"))
```

Reply:
Still haven't figured out how to implement multiline suggestions :/

Reply:
I think multiline suggestions are still not implemented by GitHub (https://github.blog/2018-11-01-suggested-changes-update/).

I rewrote it using a f-string, since we are now supporting only 3.6+